README.md

```diff
@@ -34,7 +40,7 @@ UPOS: manual native
 XPOS: not available
 Features: manual native
 Relations: manual native
-Contributors: Kahane, Sylvain
+Contributors: Pierre-Louis, Claudel, Jagodzińska, Sandra; Kahane, Sylvain; Savary, Agata; Schang, Emmanuel
```

Comment:
Apparently wrong syntax of the contributor list, which would result in a mess when published on Lindat! There are three commas in "one name", while only one is expected. How many people are these, two? And what is first name and what is last name? Should the first author be "Claudel, Pierre-Louis"?